vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -541,6 +541,8 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		Kind:        fqKindToRegister,
 
 		MetaGroupVersion: metav1.SchemeGroupVersion,
+
+		MaxRequestBodyBytes: a.group.MaxRequestBodyBytes,
```

Comment:
is this now a mix of two upstream commits? This line came from the other commit, didn't it?

Reply:
oh, this is not customresource_handler, nevermind